lib/rspec/core/reporter.rb

```diff
@@ -73,7 +73,10 @@ def start(expected_example_count, time=RSpec::Core::Time.now)
       notify :seed, Notifications::SeedNotification.new(@configuration.seed, seed_used?)
     end
 
-    # @private
+    # @api
+    # @param message to send to formatters
```

Comment:
The type specification is optional I thought? And I specifically don't want it to be `[String]` because that wouldn't support #1851